service-loadbalancer/template.cfg

```diff
@@ -4,6 +4,10 @@ global
     daemon
     stats socket /tmp/haproxy
 
+    # log using a syslog socket
+    log /var/run/haproxy.log.socket local0 info
+    log /var/run/haproxy.log.socket local0 notice
```

Comment:
What happens if we don't run the server on the other side of the socket but leave this in?